src/components/AddTaskCard/AddTaskCard.scss

```diff
@@ -0,0 +1,24 @@
+.task {
+  &__save-button {
+    background-color: $buttonPrimary;
+    border: none;
+    color: $primaryWhite;
+    cursor: pointer;
```

Comment:
Using color as a prefix or postfix in SCSS variable names can be a good practice as it helps convey the purpose or nature of the variable. It can make your code more readable and maintainable, especially when working with multiple colors or color-related variables.
```suggestion
    background-color: $colorPrimary;
    border: none;
    color: $colorWhite;
    cursor: pointer;
```